### ISSUE TITLE
Active response displays wrong status

### ISSUE THREAD

**Issue by Desvelao** [wazuh/wazuh-dashboard-plugins#6801]
**Description**

The active response status is displayed as `disabled` when it should be enabled.

Defining or not the the `disabled` property causes the same behaviour on the UI

```xml
<disabled>no</disabled>
```

the UI displays the active response is disabled.

**Screenshots**
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/34042064/2bfc669e-946b-4ad1-99c9-9502d689c2ee)

**Comment by Desvelao** [comment]
# Research

I was researching this on 4.8.0 and it seems there is a different behavior in the API responses and the expected API response by the UI.

The `Status of this active response` render depends on the `disabled` property of the API response:
- `no`: `disabled`
- any other value: `enabled`

I tried to configure the active response:

Case 1: no define `disabled` property

```xml
  <active-response>
    <command>host-deny</command>
    <location>local</location>
    <level>7</level>
    <timeout>600</timeout>
  </active-response>
```

The UI displays the status of the active response as `disabled`. The API response does not include the `disabled` property.

Case 2: define `disabled` as `no`

```xml
  <active-response>
    <disabled>no</disabled>
    <command>host-deny</command>
    <location>local</location>
    <level>7</level>
    <timeout>600</timeout>
  </active-response>
```

The UI displays the status of the active response as `disabled`. The API response does not include the `disabled` property.

Case 3: define `disabled` as `yes`

```xml
  <active-response>
    <disabled>yes</disabled>
    <command>host-deny</command>
    <location>local</location>
    <level>7</level>
    <timeout>600</timeout>
  </active-response>
```

The API response does not include the active response.

In summary, the UI is expecting the `disabled` property to render the status of the active response, but this is not included on the API response.
